src/components/MovieTile/styled.js

```diff
@@ -2,13 +2,16 @@ import { NavLink } from "react-router-dom/cjs/react-router-dom.min";
 import styled from "styled-components";
 
 export const MovieTileContent = styled.li`
+  display: flex;
+  flex-direction: column;
+  cursor: pointer;
   padding: 16px;
   border-radius: 5px;
   background-color: ${({ theme }) => theme.color.white};
   box-shadow: 0px 4px 12px 0px ${({ theme }) => theme.color.boxShadow};
   transition: 0.4s;
   width: 100%;
-  overflow: hidden;
+  gap: 16px;
```

Comment:
i think this does not work, there is no gap in tiles

Reply:
I didn't notice that. The issue arose after merging with the main branch, where additional links appeared and disrupted the layout. It's fixed now.